github/client.go

```diff
@@ -0,0 +1,2475 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
I am wondering why we just copy github API wrapper from kubernetes/test-infra? Can't we wrapper it by ourselves?

Reply:
Please note that we have already vendored https://github.com/Huawei-PaaS/ci-bot/tree/master/vendor/github.com/google/go-github and we can simply use it.